src/api/layers/WMTSLayer.class.js

```diff
@@ -48,6 +48,8 @@ export default class WMTSLayer extends AbstractLayer {
 
     /** @returns {String} A XYZ type URL to request this WMTS layer's tiles */
     getURL() {
-        return `${this.baseURL}1.0.0/${this.id}/default/${this.timeConfig.currentTimestamp}/3857/{z}/{x}/{y}.${this.format}`
+        return `${this.baseURL}1.0.0/${this.getID()}/default/${
+            this.timeConfig.currentTimestamp
+        }/3857/{z}/{x}/{y}.${this.format}`
```

Comment:
The whole viewer is based on the foundation that we will use metric WebMercator (EPSG:3857) as its projection, with on-the-fly reprojection if needed (if the user wants to see things in another projection)
We could add a default projection somewhere in the config, but I'm unsure it will be that easy to change its value.

Reply:
Ok then it make sense